### DISCUSSION TITLE
Expand Asset Folder on load

### DISCUSSION THREAD

**Discussion by alexmf23** [pimcore/pimcore#14210] [Q&A / Support]
Hello,
is there a way to open an asset folder in the tree by default when loading the page ?

![Screenshot 2023-02-01 214442](https://user-images.githubusercontent.com/63473281/216159736-3467d8b6-95e2-4786-aa30-a2b7ff854373.jpg)

**Comment by frithjof** [comment]
maybe you are better using a perspective?
